tests/perf-test/BaseOptionsList.perf-test.tsx

```diff
@@ -0,0 +1,127 @@
+import {fireEvent} from '@testing-library/react-native';
+import type {RenderResult} from '@testing-library/react-native';
+import React, {useState} from 'react';
+import {measurePerformance} from 'reassure';
+import BaseOptionsList from '@components/OptionsList/BaseOptionsList';
+import type {ListItem} from '@components/SelectionList/types';
+import variables from '@styles/variables';
+
+type BaseOptionsListWrapperProps = {
+    /** Whether this is a multi-select list */
+    canSelectMultipleOptions?: boolean;
+};
+
+describe('[BaseOptionsList] Performance tests for BaseOptionsList', () => {
```

Comment:
Can you remove `Performance tests for BaseOptionsList` to make the results easier to read